introduction.md

```diff
@@ -1 +1,6 @@
 Code School rocks! My favorite Code School path is Ruby.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?